src/shared/components/Resources/ResourceList.tsx

```diff
@@ -24,24 +30,51 @@ const ResourceList: React.FunctionComponent<ResourceListProps> = ({
   const { from, size } = paginationSettings;
   const totalPages = Math.floor(total / size);
   const current = Math.floor((totalPages / total) * from + 1);
+  const [selectedResource, setSelectedResource] = React.useState(
+    null as Resource | null
+  );
+  console.log({ selectedResource });
   return (
-    <List
-      className="resources-list"
-      loading={loading}
-      header={
-        <p className="result">{`Found ${total} resource${total > 1 && 's'}`}</p>
-      }
-      dataSource={results}
-      pagination={{
-        total,
-        current,
-        onChange: paginationChange,
-        pageSize: DEFAULT_PAGE_SIZE,
-      }}
-      renderItem={(resource: ResourceItemProps) => (
-        <ResourceItem key={resource.id} {...resource} />
+    <React.Fragment>
```

Comment:
Minor styling issue, but what do we standardize on for Fragments? Do we use the long form <React.Fragment> or the short form <> ? I like the short form personally. @julienmachon

Reply:
hmmm actually I think it's kind of an anti-pattern to use React.Fragment, I'm using it here because I'm not sure how else to attach "out-of-element" components (like modals). 

The word looks quite ugly, so it's probably better to keep it ugly to discourage its use.